test/integration/subtopic_page_test.rb

```diff
@@ -112,6 +116,10 @@ def oil_and_gas_subtopic_item(subtopic_slug, params = {})
     assert page.has_link?("Oil rig safety requirements", :href => "/oil-rig-safety-requirements")
     assert page.has_link?("North sea shipping lanes", :href => "/north-sea-shipping-lanes")
     assert page.has_link?("Undersea piping restrictions", :href => "/undersea-piping-restrictions")
+
+    # The following selector is specified using XPath because Capybara/Nokogiri does not seem to find non-standard tags
+    # using the usual "CSS-style" selectors.
+    assert page.has_selector?(:xpath, "//test-govuk-component[@data-template='govuk_component-breadcrumbs']", visible: true)
```

Comment:
Should this be using the shared test helpers that slimmer provides?
eg https://github.com/alphagov/slimmer/blob/master/lib/slimmer/test_helpers/govuk_components.rb

Example of it being used in government-frontend:
https://github.com/alphagov/government-frontend/blob/master/test/test_helper.rb#L57-L61